unittests/emitc_tosa.cpp

```diff
@@ -27,11 +27,11 @@ const float EPSILON = 5e-4;
 // Unary elementwise ops
 TEST(tosa, reciprocal) {
   Tensor<float, 4> t0{1.0f, 2.0f, 3.0f, 4.0f};
-  Tensor<float, 4> excpected{1.0f, 0.5f, 0.3333f, 0.25f};
+  Tensor<float, 4> expected{1.0f, 0.5f, 0.3333f, 0.25f};
 
   Tensor<float, 4> result = tosa::reciprocal(t0);
 
-  EXPECT_THAT(result, Pointwise(FloatNear(EPSILON), excpected));
+  EXPECT_THAT(result, Pointwise(FloatNear(EPSILON), expected));
```

Comment:
Uh, a previous bug?

Reply:
Nope, just a typo.

Reply:
And in this PR as well, meaning you'll creating a merge conflict? ;)

Reply:
Nope, I mixed the PRs... I have fixed it here! Too much code :rofl:

Reply:
You may want to add a short note that this further fixes a unittest to the PR descripton, which we adopt as commit message (I am certain we will squash this one :wink:) . No need for a separate PR though.